pages/09.education-team/chapter.md

```diff
@@ -10,6 +10,15 @@ The Education Team is responsible for the Forum, Mautic Documentation and Intern
 
 > To improve the resources available for new and established users to learn how to use Mautic
 
+### General Tasks
+
+  * Help marketers to contribute their existing content to the Mautic knowledge base
```

Comment:
```suggestion
  * Help marketers to contribute their existing content to the [Mautic Knowledgebase](https://kb.mautic.org)
```

So far we have used all one word and capitalised - not sure if that is correct, maybe we should use two separate words! :o